draft-ietf-lamps-kyber-certificates.md

```diff
@@ -279,7 +279,9 @@ algorithm itself.
 </aside>
 
 When used in a OneAsymmetricKey type, the privateKey OCTET STRING contains
-the raw octet string encoding of the 64-octet seed.
+the raw octet string encoding of the 64-octet seed. The publicKey field
```

Comment:
Perhaps also say that the OneAsymmetricKey version field SHOULD be set to 0 (v1).